fe/src/components/textbox/TextBox.tsx

```diff
@@ -41,10 +41,6 @@ export default function TextBox(props: TextBoxProps) {
   const handleKeyDown: React.DOMAttributes<HTMLDivElement>["onKeyDown"] = (
     event
   ) => {
-    if (event.key === "Control") {
-      event.preventDefault();
-      handleToggleType();
-    }
 
```

Comment:
There is potentially an additional new line here that I would vote to have removed, but for a first submission I'll let it slide ;).  I'll eventually get instructions up for auto formatting on save, which should automatically deal with white space in a uniform way.

Reply:
delete line 44?